src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.cs

```diff
@@ -102,6 +102,17 @@ public void FileVersionInfo_NonExistentFile_FileNotFound()
                 FileVersionInfo.GetVersionInfo(Path.Combine(Directory.GetCurrentDirectory(), TestNotFoundFileName)));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Don't want to create temp file in app container current directory")]
+        public void FileVersionInfo_RelativePath_CorrectFilePath()
+        {
+            using (new FileStream("kernelbase.dll", FileMode.CreateNew, FileAccess.ReadWrite, FileShare.None, 0x1000, FileOptions.DeleteOnClose))
+            {
+                FileVersionInfo fvi = FileVersionInfo.GetVersionInfo("kernelbase.dll");
+                Assert.Equal(Path.GetFullPath("kernelbase.dll"), fvi.FileName);
```

Comment:
This check of the FileName property is tricky. It was not obvious to me at first  what it is doing. It deserves a comment.

Or instead - it may better to check that some of the properties are empty (like FileDescription or Product). They are non-empty for kernelbase.dll that comes with Windows.

Reply:
I will add a check for FileDescription in addition to the FileName check.